src/pages/index.tsx

```diff
@@ -26,6 +26,11 @@ const IndexPage = ({ data }) => {
     <Layout>
       <Heading as="h1" srOnly>{blok.title || 'Homepage'}</Heading>
       <HomepageHero />
+      <img
+        alt=""
+        src={getProcessedImage('https://a-us.storyblok.com/f/1005200/2000x40/f23b53c0e4/steve-johnson-cropped-2000x40-02.jpg')}
+        className="su-w-full"
+      />
```

Comment:
These are divider bars with background textured images. Will be extracted into a component later. Currently inserting them in places that Kerri requested and see if clients like them